apps/ui/src/networks/offchain/index.ts

```diff
@@ -1,14 +1,19 @@
 import { createApi } from './api';
 import * as constants from './constants';
 import { pinPineapple } from '@/helpers/pin';
-import { Network } from '@/networks/types';
-import { NetworkID } from '@/types';
+import { Network, VotingPower } from '@/networks/types';
+import { NetworkID, StrategyParsedMetadata } from '@/types';
 
 const HUB_URLS: Partial<Record<NetworkID, string | undefined>> = {
   s: 'https://hub.snapshot.org/graphql',
   's-tn': 'https://testnet.hub.snapshot.org/graphql'
 };
 
+const SCORE_URLS: Partial<Record<NetworkID, string | undefined>> = {
+  s: 'https://score.snapshot.org',
+  's-tn': 'https://score.snapshot.org'
+};
```

Comment:
Do they actually depend on the NetworkID? (they are the same here) - if it's just single instance let's have single URL.